README.md

```diff
@@ -79,6 +79,20 @@ These are the members of [@open-telemetry/docs-approvers]:
 For more information about the approver role, see the
 [community repository](https://github.com/open-telemetry/community/blob/main/community-membership.md#approver).
 
+### Triagers
+
+These are the members of [@open-telemetry/docs-triagers]:
+
+- [Emídio Neto](https://github.com/emdneto)
+- [Ezzio Moreira](https://github.com/EzzioMoreira), PicPay
+- [Kazunori Otani](https://github.com/katzchang), Splunk
+- [Masaki Sugimoto](https://github.com/Msksgm)
```

Comment:
I confirmed the official company name in English. 
`Henry, Inc.` requires `, ` between `Henry` and `Inc.`.

```suggestion
- [Masaki Sugimoto](https://github.com/Msksgm), Henry, Inc.
```